webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9628,6 +9627,35 @@ var userOverride = {
                 return '';
             };
 
+            
+            _this.getMessageCreatedAtTime = function (createdAtTime) {
+                if (TIME_FORMAT_24HOURS) {
+                    
+                    var month = ["Jan", "Feb", "Mar", "April", "May", "June", "July", "Aug", "Sep", "Oct", "Nov", "Dec"];
```

Comment:
do we need to hard code these? there might be some function in javascript to get the month name based on the month number
how will the localization work? do we handle translation to other language in existing format?

Reply:
Removed the hardcoding. and used toLocaleString to get the month name.